prov/rxm/src/rxm_ep.c

```diff
@@ -921,6 +921,7 @@ rxm_ep_sar_tx_prepare_segment(struct rxm_ep *rxm_ep, struct rxm_conn *rxm_conn,
 	tx_buf->pkt.ctrl_hdr.seg_no = seg_num;
 	tx_buf->pkt.ctrl_hdr.seg_size = seg_len;
 	tx_buf->pkt.ctrl_hdr.segs_cnt = tx_entry->segs_left;
+	tx_buf->pkt.hdr.flags |= comp_flags & (FI_MSG | FI_TAGGED);
```

Comment:
`comp_flags` is neither `FI_MSG` nor `FI_TAGGED`.
the `comp_flags` is passed as an argument to the `rxm_ep_send/inject_common`.

We can just make the following:
`tx_buf->pkt.hdr.flags |= comp_flags;`

Reply:
> comp_flags is neither FI_MSG nor FI_TAGGED.

Did you mean either ... or?

I'll add an assert to make sure we're right (in case we change what can be passed in `comp_flags` in the future).

Reply:
Sure, let me add in other places as well.